react/features/remote-video-menu/components/MuteRemoteParticipantDialog.web.js

```diff
@@ -77,18 +77,12 @@ class MuteRemoteParticipantDialog extends Component {
      * Handles the submit button action.
      *
      * @private
-     * @returns {void}
+     * @returns {boolean} - True.
```

Comment:
Maybe also add true is to note that the modal should be closed?

Reply:
👍